rust/connlib/snownet/tests/lib.rs

```diff
@@ -1,37 +1,70 @@
 use boringtun::x25519::{PublicKey, StaticSecret};
+use firezone_relay::{AddressFamily, AllocationPort, ClientSocket, IpStack, PeerSocket};
+use rand::rngs::OsRng;
 use snownet::{Answer, ClientNode, Event, MutableIpPacket, ServerNode, Transmit};
 use std::{
     collections::HashSet,
     iter,
     net::{IpAddr, Ipv4Addr, SocketAddr, SocketAddrV4},
-    time::{Duration, Instant},
+    time::{Duration, Instant, SystemTime},
     vec,
 };
 use str0m::{net::Protocol, Candidate};
-use tracing::{info_span, Span};
+use tracing::{debug_span, info_span, Span};
 
 #[test]
-fn smoke() {
+fn smoke_direct() {
     let _ = tracing_subscriber::fmt()
         .with_test_writer()
         .with_env_filter("debug")
         .try_init();
 
     let (alice, bob) = alice_and_bob();
 
-    let mut alice = TestNode::new(info_span!("Alice"), alice);
-    alice.add_local_host_candidate("1.1.1.1:80");
+    let mut alice =
+        TestNode::new(info_span!("Alice"), alice, "1.1.1.1:80").with_local_host_candidate();
```

Comment:
Is 1.1.1.1 Alice's IP or the Cloudflare DNS?

Reply:
Just Alice's IP. Can be anything :)